### PR TITLE
LPS-80771 Create poshi test to test the case of changing a field type of numeric field into calculator expression

### PR DESCRIPTION
**Pull request by alinedoleron**
Related Issue: https://issues.liferay.com/browse/LPS-80771

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase [hunk 1]
```diff
@@ -35,6 +35,73 @@ definition {
...
+			conditionOperator = "Is equal to",
+			conditionRowNumber = "1",
+			conditionValueType = "Value");
+
+		FormRules.configureActionOptions(
+			action = "Require",
+			actionRowNumber = "1",
+			actionTarget = "Text Field 2");
+
+		FormsAdminNavigator.gotoRules();
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormRules.testcase:84]: @alinedoleron, please see https://github.com/natocesarrego/liferay-portal/pull/905#discussion_r522092659.

**Review by natocesarrego — CHANGES_REQUESTED** [review]

**Pull request closed without merging**